task1_weatherman/weatherman.py

```diff
@@ -0,0 +1,90 @@
+import os
+import csv
+import argparse
+import sys
+
+class Weathermachine:
```

Comment:
Let's create two classes. One to handle file reading and data cleaning and other to perform data analysis.